manageiq-gems-pending.gemspec

```diff
@@ -37,6 +37,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "ftpd",                      "~> 2.1.0"
   s.add_development_dependency "manageiq-style",            ">= 1.5.4"
 
+  s.add_development_dependency "rack",                      "~> 3.1.16" # this ensures manageiq-style's rack requirement is safe
```

Comment:
I thought manageiq core didn't work with rack 3 - maybe put another way - when I bundle update in core I get rack 2

Reply:
This is a development dependency.

This `rack` requirement is only installed when `bundle install` inside `manageiq-gems-pending`. These dependencies are not transient to other projects.

When `bundle update` in `manageiq` core, it will ignore the `rack` requirement from here.
Of course, it will get a def rack requirement from `manageiq-style`, and a runtime requirement from rails. I assume mend won't be too happy in other scanned projects with no `rack` requirement.